appcli/orchestrators.py

```diff
@@ -101,24 +119,47 @@ class DockerComposeOrchestrator(Orchestrator):
 
     def __init__(
         self,
-        docker_compose_file: Path,
-        docker_compose_override_files: Iterable[Path] = [],
+        docker_compose_file: Path = Path("docker-compose.yml"),
+        docker_compose_override_directory: Path = Path("docker-compose.override.d/"),
+        docker_compose_task_file: Path = Path("docker-compose.tasks.yml"),
+        docker_compose_task_override_directory: Path = Path(
+            "docker-compose.tasks.override.d/"
+        ),
     ):
         """
-        Creates a new instance.
+        Creates a new instance of an orchestrator for docker-compose-based applications.
 
         Args:
-            docker_compose_file (Path): Path to a `docker-compose.yml` file relative to the generated configuration directory.
-            docker_compose_override_files (Iterable[Path], optional): Paths to any additional docker-compose override files relative to the generated configuration directory.
+            docker_compose_file (Path): Path to a docker compose file containing long-running services. Path is relative
+                to the generated configuration directory.
+            docker_compose_override_directory (Path, optional): Path to a directory containing any additional
+                docker-compose override files. Overrides are applied in alphanumeric order of filename. Path is relative
+                to the generated configuration directory.
+            docker_compose_task_file (Path): Path to a docker compose file containing services to be run as short-lived
+                tasks. Path is relative to the generated configuration directory.
+            docker_compose_task_override_directory (Path): Path to a directory containing any additional
+                docker-compose override files for services used as tasks. Path is relative to the generated
+                configuration directory.
         """
         self.docker_compose_file = docker_compose_file
-        self.docker_compose_override_files = docker_compose_override_files
+        self.docker_compose_override_directory = docker_compose_override_directory
+        self.docker_compose_task_file = docker_compose_task_file
+        self.docker_compose_task_override_directory = (
+            docker_compose_task_override_directory
+        )
```

Comment:
Why the brackets?

Reply:
Autoformatting by `Black`. Since the line will be too big, it splits it onto the next line.